CHANGELOG.md

```diff
@@ -1,8 +1,27 @@
-# dbt_youtube_analytics_source v0.UPDATE.UPDATE
+# dbt_youtube_analytics_source v0.4.0
 
```

Comment:
Just [the same comment as the transform](https://github.com/fivetran/dbt_youtube_analytics/pull/13#discussion_r1424225513) here.

Reply:
Thanks, just updated